### DISCUSSION TITLE
How to use

### DISCUSSION THREAD

**Discussion by DragonHunter274** [Guekka/xdg-desktop-portal-termfilepickers#1] [Q&A]
I installed termfilepickers using the module in the termfilechooses nixpkgs PR, but cant get the yazi file picker to show up, the default (gtk?) file picker always shows up

EDIT: just have to tell the apps to use the portal, works great now
Now I just have to figure out how to get the window to open floating
and to get qt apps to use the portal

EDIT2: actually it just works with firefox nothing else

**Comment by Guekka** [comment]
Sorry for the delay, I did not get the notification
Please have a look at the readme from here https://github.com/boydaihungst/xdg-desktop-portal-termfilechooser

Feel free to ping me
